### DISCUSSION TITLE
German translation: form of address formal or informal

### DISCUSSION THREAD

**Discussion by CFM90** [apache/superset#18671] [Q&A / Help] [answered]
I am currently working on a german translation of Apache Superset. I was wondering if you prefer the formal or informal way of addressing users. Some indications from a working environment: 

Formal way: "Sie"
- Blue Chip style working environment
- Sounds more professional
- Sounds more distant and old 

informal way: "Du"
- Start up style working environment
- Sounds younger
- sounds closer

I am clearly biased torwards "Du" because the tool is fresh, young and nothing like a Blue Chip Monster. 

But I would like to heare some feedback.

**Comment by ad-m** [comment, marked as answer]
Do you have data on how to deal with this issue competing tools to Superset? There seems to be no reason for us to be different, but to follow the flow.

**Reply by CFM90** [reply]
Power BI uses the formal form "Sie":
![grafik](https://user-images.githubusercontent.com/24831579/153558844-d2a69896-ca74-4c01-89ea-242c179a417e.png)

And judged by the Website, Tableau uses the formal form too. 
![grafik](https://user-images.githubusercontent.com/24831579/153559029-93b9e6e0-4491-400d-bcbd-2f988dcf8afa.png)

**Reply by villebro** [reply]
I agree with @CFM90 , it feels like the most appropriate form. Ping @hbruch who has done a lot of work on the German translation of Superset.

**Reply by hbruch** [reply]
I used the more formal 'Sie' which I find more appropriate in a business context, which IMHO superset targets more. So I agree with @villebro and @ad-m . Thanks @CFM90 for raising the question and checking Tableau and PowerBI.
